### PR TITLE
Plugin for Docker containers.

### PR DESCRIPTION
**Pull request by johann8384**
Testing

### CODE REVIEW

**Comment by johann8384** [comment]
Rebase of original PR #209 submitted by @mdavranche

**Comment by johann8384** [comment]
This still needs some work, I will probably move it to the next release.

#### Current metrics:
- docker.blkio.io_service_bytes.read
- docker.blkio.io_service_bytes.write
- docker.memory.active_anon
- docker.memory.active_file
- docker.memory.cache
- docker.memory.hierarchical_memory_limit
- docker.memory.hierarchical_memsw_limit
- docker.memory.inactive_anon
- docker.memory.inactive_file
- docker.memory.mapped_file
- docker.memory.rss
- docker.memory.swap
- docker.memory.unevictable

I will probably refactor those to be docker.blkio.io_service_bytes with type=read/write and docker.memory with type={all that stuff}

**Comment by mdavranche** [comment]
Nice! Do not hesitate if you need help on this issue.

File: collectors/etc/docker_conf.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# of the GNU Lesser General Public License along with this program.  If not,
+# see <http://www.gnu.org/licenses/>.
+
+def enabled():
+  return False
+
+def get_config():
+  config = {
+    'interval': 15,
+    'cgroup_path': '/sys/fs/cgroup'
```
Comment (vitaliyf) [inline collectors/etc/docker_conf.py]: Would be nice to mention that this should be "/cgroup" on CentOS/RHEL6.

**Pull request merged**